modules/command/dh/dh-litigation-answered-command.js

```diff
@@ -44,7 +46,34 @@ class DHLitigationAnsweredCommand extends Command {
                 return Command.empty();
             }
         }
-        return Command.repeat();
+        return Command.retry();
```

Comment:
This command should be repeating since it's waiting for a blockchain event, but we should add a deadline_at (just like on DCLitigationInitiatedCommand)